cmd/api/src/api/middleware/middleware.go

```diff
@@ -93,113 +72,37 @@ func getScheme(request *http.Request) string {
 	}
 }
 
-const (
-	// Default timeout for any request is thirty seconds
-	defaultTimeout = 30 * time.Second
-
-	// This is set to 1 minute since intermediate load balancers may terminate requests that sit longer than 1 minute
-	maximumTimeout = time.Minute
-)
-
 // ContextMiddleware is a middleware function that sets the BloodHound context per-request. It also sets the request ID.
 func ContextMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(response http.ResponseWriter, request *http.Request) {
 		var (
 			startTime = time.Now()
-			requestID string
+			requestID = "ERROR"
```

Comment:
Is there a reason to initialize requestID here? It seems like there's never a code path that will activate with it set to `ERROR` and if it did go off the rails, the zero value should be sufficient

Reply:
Not really. I'll revert this. It was more just my preference.

Reply:
Reverted :)

Reply:
Oh, I misread the error path. For some reason I read the `log.Errorf()` and assumed that was the end of the story. I still like this approach, but now it makes sense. I think either way it's written is fine now that I'm actually reading it properly.